Application/Service/DataGateway.php

```diff
@@ -18,7 +18,7 @@
 class DataGateway
 {
     /**
-     * @var EntityManager
+     * @var EntityManagerInterface
```

Comment:
It is not better to set these hints in a single line?.

Reply:
We're using multiline comments for `@var` everywhere in this project. So if we change it we must change them all